src/web/mjs/connectors/WeebCentral.mjs

```diff
@@ -0,0 +1,84 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class WeebCentral extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'weebcentral';
+        super.label = 'Weeb Central';
+        this.tags = [ 'manga', 'manhua', 'manhwa', 'english' ];
+        this.url = 'https://weebcentral.com';
+    }
+
+    async _getMangas() {
+        const limit = 32;
+        let offset = 0;
+        let mangaList = [];
+        let run = true;
+
+        while (run) {
+            const uri = new URL(`${this.url}/search/data?display_mode=Minimal+Display&limit=${limit}&offset=${offset}`);
+
+            const request = new Request(uri, this.requestOptions);
+            const data = await this.fetchDOM(request, 'article.bg-base-300');

```

Comment:
```article.bg-base-300``` doesnt seems to be a good CSS selector. 'bg-base-300' seems to be autogenerated class name based on style.  Better doing it like in Haruneko https://github.com/manga-download/haruneko/blob/master/web/src/engine/websites/WeebCentral.ts#L14